README.md

```diff
@@ -36,13 +36,20 @@ Given the complexity of both C++ and Rust, it is unlikely that this will
 ever be fixed, and there are no plans to do so. Of course, patches to
 fix specific consensus incompatibilities are welcome.
 
+### Support for 16-bit pointer sizes
```

Comment:
Nit: The limitation is that there is _no_ support.

```suggestion
### No support for 16-bit pointer sizes
```

Reply:
I went for consistency with previous chapter ("Consensus", not "No consensus").